remote.go

```diff
@@ -44,6 +44,16 @@ var remoteErrors = map[int]string{
 	32: "invalid selector",
 }
 
+// Alias for a type that is passed as an argument for selenium.Wait
+type Condition func(wd WebDriver) (bool, error)
```

Comment:
Move this type definition and the associated constants to just before the WaitWithTImeoutAndInterval function definition, so that all related identifiers are together.